### PR TITLE
Dev/ignition 8.1 1.2

### PR DESCRIPTION
**Pull request by alexjhawk**

### CODE REVIEW

**Comment by TomKimsey** [comment]
made one pass through and nothing caught my eye. Ill give it another look after lunch.

File: eWonConnector-build/pom.xml [hunk 1]
```diff
@@ -48,7 +48,7 @@
                     <moduleName>${project.parent.name}</moduleName>
                     <moduleDescription>${project.description}</moduleDescription>
                     <moduleVersion>1.1.12</moduleVersion>
-                    <requiredIgnitionVersion>8.0.0</requiredIgnitionVersion>
+                    <requiredIgnitionVersion>8.1.0</requiredIgnitionVersion>
```
Comment (TomKimsey) [inline eWonConnector-build/pom.xml:51]: Curious as to why the required ignition version is less than the sdk version. is  sdk v8.1.1 supported in v8.1.0?

**Review by TomKimsey — APPROVED** [review]

File: eWonConnector-build/pom.xml (hunk 1 above)
Comment (alexjhawk) [reply eWonConnector-build/pom.xml]: Not sure, but I just made the required version the same as the SDK version

File: eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/comm/AuthInfo.java [hunk 2]
```diff
@@ -146,11 +146,20 @@ public String getEwonPassword() {
...
 
   /**
-   * Generate an HTTP GET string with authentication information
+   * Generate a string with authentication information for use with the DMWeb API on Talk2M.
    *
-   * @return authentication information HTTP GET string
+   * @return DMWeb API authentication information string
    */
-  public String toGetString() {
+  public String toDMPostString() {
```
[the remark this answers is not in the document]
Comment (alexjhawk) [reply eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/comm/AuthInfo.java:153]: Added to commit message

**Review by oliver-walker-hms — APPROVED** [review]

**Review by TomKimsey — APPROVED** [review]

**Pull request merged**